src/Factories/FunFunFactory.php

```diff
@@ -852,7 +852,11 @@ public function getDonationConfirmationPageSelector() {
 	}
 
 	public function newDonationFormViolationPresenter() {
-		return new DonationFormViolationPresenter( $this->getLayoutTemplate( 'DonationForm.html.twig' ) );
+		$template = $this->getLayoutTemplate( 'DisplayPageLayout.twig' );
+		// TODO make this dependent on the 'form' value from the HTTP POST request
+		// (we need different form pages for A/B testing)
```

Comment:
We have stuff in place already for showing different donation confirmation pages. There no HTTP parameter is used. Does that make sense?